components/BackupAndRestoreAllScreenBanner.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {useBackupScreen} from '../screens/backupAndRestore/BackupController';
+import {BannerNotification} from './BannerNotification';
+import {Theme} from './ui/styleUtils';
+import {useTranslation} from 'react-i18next';
+import {useBackupRestoreScreen} from '../screens/Settings/BackupRestoreController';
+
+export const BackupAndRestoreAllScreenBanner: React.FC = () => {
+  const backUpController = useBackupScreen();
+  const restoreController = useBackupRestoreScreen();
+
+  const {t} = useTranslation('BackupAndRestoreBanner');
+
+  function backupFailure() {
+    const translation = t(
+      `backupFailure.${backUpController.backupErrorReason}`,
+    );
+
+    return (
+      <BannerNotification
+        message={translation}
+        onClosePress={backUpController.DISMISS}
+        key={`backupFailure-${backUpController.backupErrorReason}`}
+        testId={`backupFailure-${backUpController.backupErrorReason}`}
+        customStyle={Theme.Styles.dataBackupFailure}
+      />
+    );
+  }
+
+  function restoreFailure() {
+    const translation = t(
+      `restoreFailure.${restoreController.restoreErrorReason}`,
+    );
+
+    return (
+      <BannerNotification
+        key={`restoreFailure-${restoreController.restoreErrorReason}`}
+        message={translation}
+        onClosePress={restoreController.DISMISS}
+        testId={`restoreFailure-${restoreController.restoreErrorReason}`}
+        customStyle={Theme.Styles.dataBackupFailure}
+      />
+    );
+  }
+
+  return (
+    <>
```

Comment:
Is this line required?

Reply:
Yes we need fragment here to wrap children